server/game/cards/01-Core/OldBearMormont.js

```diff
@@ -11,24 +11,16 @@ class OldBearMormont extends DrawCard {
             when: {
                 onPhaseEnded: event => event.phase === 'challenge' && this.controller.getNumberOfChallengesLost('defender') === 0
             },
-            handler: () => {
-                this.game.promptForSelect(this.controller, {
-                    cardCondition: card => card.location === 'hand' && card.controller === this.controller && card.isFaction('thenightswatch'),
-                    activePromptTitle: 'Select character',
-                    source: this,
-                    onSelect: (player, card) => this.onCardSelected(player, card)
-                });
+            target: {
```

Comment:
This needs a non-default 'Select a card' prompt, as any NW card can be put into play from hand.